Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -744,6 +742,47 @@ def export_request_with_export_uuid(export_uuid: str, assets_or_vulns: str) -> d
     return res.json()
 
 
+def get_scanner_info():
+    """
+    Returns the scanner list and additional information about each scanner
+    Args:
+        NEW_HEADERS and USE_SSL
+    Returns:
+        CommandResults: Command object with relevant data.
+    """
+    full_url = f'{BASE_URL}scanners'
+    try:
+        response = requests.get(full_url, headers=NEW_HEADERS, verify=USE_SSL)
+        if response.status_code != 200:
+            raise DemistoException(response.text)
```

Comment:
```suggestion
        response.raise_for_status()
```
(unless we want to check specifically for a `200` response, and other responses like `201`, `202`, etc are considered invalid for some reason)